resource/file.go

```diff
@@ -37,3 +40,8 @@ func (f *File) Info() (os.FileInfo, error) {
 	}
 	return fi, nil
 }
+
+// Implements Stringer interface
+func (f *File) String() string {
```

Comment:
If this is `Stringer` interface we should move this method out from `Filer` interface.